src/skai/model/inference_lib.py

```diff
@@ -1,15 +1,21 @@
 """Functions for running model inference in beam."""
 
 import time
-from typing import Any, Iterator
-
+from typing import Any, Iterator, List, Dict
 import apache_beam as beam
 import apache_beam.utils.shared as beam_shared
 import numpy as np
 from skai import utils
 from skai.model import data
 import tensorflow as tf
 
+_GPUS = tf.config.list_physical_devices('GPU')
```

Comment:
Can we turn this into an initialization function that gets called right after its definition? Just to avoid running code at module level.

Reply:
Sounds good to me!